### ISSUE TITLE
Touchpad two finger tap

### ISSUE THREAD

**Issue by Miosame** [AngelLopez2/X250-Hackintosh#1]
Is there any way to get two finger touch to recognize with VoodooPS2Controller? or another kext that could map the touchpad in the X250 like voodoo does in the addition to that?

**Comment by Miosame** [comment]
TL;DR: download and install this kext: https://github.com/acidanthera/bugtracker/issues/418#issuecomment-547674864

---

According to a [reddit user](https://www.reddit.com/r/hackintosh/comments/dge6qt/thinkpad_t450s_mojave_trackpad_gestures_possible/f3b90fb) Rehabman (the one linked in this repo) is choppy and not as great as acidanthera - but acidantheras build does not work with the touchpad if you install it, what does work is the randomly linked kext here: https://github.com/acidanthera/bugtracker/issues/418#issuecomment-547674864

---

Would there be a way to make upstream acidanthera work with this repo?
